README.md

```diff
@@ -1,2 +1,79 @@
-# CESM GMOM_JRA configuration
-CESM GMOM_JRA configuration (MOM6-CICE6-DATM-DROF-SLND-SWAV-SGLC) to run with payu
+# MOM6-CICE6-datm-drof configurations for ACCESS-OM3
+
+This repository contains several
+[ACCESS-OM3](https://github.com/COSIMA/access-om3) configurations using the following components:
+
+- [MOM6](https://github.com/mom-ocean/MOM6)
+- [CICE](https://github.com/CICE-Consortium/CICE)
```

Comment:
```suggestion
- [CICE](https://github.com/ESCOMP/CICE)
```

Reply:
Should the link be CICE6?

Reply:
It seems that on the CICE repo, developers always refer to the model as CICE, not CICE6, so I tried to respect that choice. This is in contrast to MOM6.